scripts/ci/check_compliance.py

```diff
@@ -1084,6 +1087,48 @@ def run(self):
             except MaintainersError as ex:
                 self.failure(f"Error parsing {file}: {ex}")
 
+class ModulesMaintainers(ComplianceTest):
+    """
+    Check that all modules have a MAINTAINERS entry.
+    """
+    name = "ModulesMaintainers"
+    doc = "Check that all modules have a MAINTAINERS entry."
+    path_hint = "<git-top>"
```

Comment:
This makes it look like the check could be run in any repository, not just the zephyr repository, but that doesn't quite make sense with what's going on below.

Reply:
Does it? Not sure it would make much sense as it is now but I certainly wouldn't prevent it, may need some extra check, guess it can be added if someone comes up with a use case for it.